### PR TITLE
Edit questions patch

### PR DESCRIPTION
**Pull request by awqmayo**
Add editing functionality to the tradeoff and sum questions.

### CODE REVIEW

File: src/components/TabList/TabList.jsx [hunk 1]
```diff
@@ -145,6 +145,7 @@ class TabList extends Component {
             })
             break;
           case "tradeoff":
+            console.log("Are we even not here?");
```
Comment (brownsarahm) [inline src/components/TabList/TabList.jsx:148]: ?

Comment (awqmayo) [reply src/components/TabList/TabList.jsx]: Sorry, I was trying to figure out where the error was being thrown and got exasperated :). I deleted it.

Comment (brownsarahm) [reply src/components/TabList/TabList.jsx]: it looked like debugging, but I fgured it was no longer needed.

**Comment by Elvictor01** [comment]
@awqmayo @brownsarahm The edit function now works for the TradeOff App but there is an error in the SumQuestion module

Attached a screenshot below for some context with the SumQuestion module
SumQuestion Error

**Comment by awqmayo** [comment]
> @awqmayo @brownsarahm The edit function now works for the TradeOff App but there is an error in the SumQuestion module
> 
> Attached a screenshot below for some context with the SumQuestion module SumQuestion Error

I will take a look @Elvictor01 @brownsarahm , thank you

**Comment by awqmayo** [comment]
@Elvictor01 I think I have fixed the sum question editing bug; please take a look when you get a chance. 

@brownsarahm

**Comment by Elvictor01** [comment]
> @Elvictor01 I think I have fixed the sum question editing bug; please take a look when you get a chance.
> 
> @brownsarahm

I would check it out later today. Thanks.

**Comment by Elvictor01** [comment]
Both the TradeOff and SumQuestion no longer work when attempted to edit @awqmayo @brownsarahm 


ErrorSQ
ErrorTrOff

**Comment by awqmayo** [comment]
Taking a look tonight thank you @Elvictor01

**Comment by awqmayo** [comment]
@Elvictor01 If the experiment configurations don't contain data, the edit question function won't work because it requires the researcher to have submitted data. 

What JSON files did you use to test these? For the tradeoff app when I look at the experiment configs it's not showing data for the bar graph heights which need to be sent to the edit experiment app on the back end.

**Comment by Elvictor01** [comment]
> @Elvictor01 If the experiment configurations don't contain data, the edit question function won't work because it requires the researcher to have submitted data.
> 
> What JSON files did you use to test these? For the tradeoff app when I look at the experiment configs it's not showing data for the bar graph heights which need to be sent to the edit experiment app on the back end.

@awqmayo thanks for pointing my attention to it, I apparently mixed up data files while working with it.

**Comment by Elvictor01** [comment]
@brownsarahm TradeOff and SumQuestion are both editable now!

**Review by Elvictor01 — APPROVED** [review]

**Pull request merged**